### ISSUE TITLE
Lualine support for SpaceCamp?

### ISSUE THREAD

**Issue by vzytoi** [jaredgorski/SpaceCamp#21]
Hi guys, any plan about a [lualine ](https://github.com/hoob3rt/lualine.nvim) support for SpaceCamp?

**Comment by jaredgorski** [comment]
I've avoided accepting or building too many 3rd party integrations into the codebase directly, since I think it's more elegant to create separate SpaceCamp themes for those integrations which users can install separately. Fortunately, it looks like `lualine` has implemented their own theme repository [here](https://github.com/hoob3rt/lualine.nvim/tree/master/lua/lualine/themes). That's perfect, since it lets me keep the SpaceCamp package devoted mostly just to syntax.

They have [documentation for Contributing](https://github.com/hoob3rt/lualine.nvim/blob/master/CONTRIBUTING.md) in case someone would like to add a theme. Since I don't personally use `lualine` (or `airline`, or any of the others), I haven't considered writing a theme for those. However, if you create a theme for `lualine` (or anything else) and it gets accepted, I'll gladly list your contribution in the [Outside of Vim](https://github.com/jaredgorski/SpaceCamp#----outside-of-vim--) section on the SpaceCamp README so that other users can easily find it.

It does [look like they're accepting new colorschemes](https://github.com/hoob3rt/lualine.nvim/pulls?q=is%3Apr+is%3Aclosed), since the last addition was merged less than 2 weeks ago, so that's good news 😸
